examples/diagnostics_and_criticism/sampler-stats.ipynb

```diff
@@ -14,58 +14,56 @@
   },
```

Comment:
<p>I would use <code>trace.sample_stats["diverging"].sum()</code>  to see if there are divergences, if there were, we can select them using the diverging variable as a mask _independently of the dimensions of the array we want to subset_. Something like this should get the samples in the posterior that correspond to diverging iterations</p><p> </p><pre class="ql-syntax" spellcheck="false">trace.posterior.where(trace.sample_stats["diverging"])
</pre><p> </p><p>for all variables at once.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/94/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>